hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -691,6 +702,13 @@ onOpenNetworkReqSn env ledger st otherParty sn requestedTxs =
       then continue
       else Wait $ WaitOnSnapshotNumber seenSn
 
+  notSeenTxs = fromList (txId <$> requestedTxs) `Set.difference` fromList (txId <$> seenTxs)
+
+  waitSeenTxs continue =
+    if null notSeenTxs
```

Comment:
Could: write this more in line to the math notation using [`isSubsetOf`](https://hackage.haskell.org/package/containers-0.6.7/docs/Data-Set.html#v:isSubsetOf)

 ```suggestion
    if fromList requestedTxs `isSubsetOf` fromList seenTxs
```

Reply:
Of course 🤦